src/components/ApiUsage.jsx

```diff
@@ -97,27 +113,11 @@ const ApiUsage = (props:tPROPS) => {
       this.onTrackerChanged = this.onTrackerChanged.bind(this)
     }
 
-    componentDidMount () {
+    componentDidMount = () => {
```

Comment:
This and other similar function declaration changes aren't really necessary. Were you getting errors from your IDE? Set the JavaScript version as follows:
![image](https://user-images.githubusercontent.com/11096575/102778700-1057a480-4361-11eb-8605-f673bb17ff0f.png)

Reply:
sure Denis, I see the errors in the IDE and also this blog says to use the arrow functions. 
https://codeburst.io/how-to-not-react-common-anti-patterns-and-gotchas-in-react-40141fe0dcd

Reply:
The biggest attraction to using arrow functions like mentioned in this article is that you don't need to bind them in the constructor any more. Make sure you remember to remove the binding statements in the constructor after converting, I still see some there

Reply:
The biggest draw to converting to arrow functions is that they no longer need to be bound in the constructor. You've converted a few like onHighlightChange(), make sure you remember to remove them from the constructor as well, there are a few still there.

That being said, life-cycle functions like componentDidMount should **not** be converted, those should stay a traditional function declaration: componentDidMount ()

Reply:
👍

Reply:
I'll make these changes, Thanks.